source/reflect.h2

```diff
@@ -1676,8 +1677,8 @@ parse_context_group_state: @struct type =
         _ = alternate_match_lists..insert(alternate_match_lists..end(), shared.new<regex_token_list>(new_list));
     }
 
-    //  Swap this state with the other one.
-    swap: (inout this, inout t: parse_context_group_state) = {
+    //  Swap this state with the other one. NOLINTNEXTLINE(performance-noexcept-swap)
+    swap: (inout this, inout t: parse_context_group_state) = { // NOLINT(performance-noexcept-swap)
```

Comment:
This smells like it might be `noexcept`.

Reply:
Definitely, tell me how to write it and I will change it, it should be noexcept.

Reply:
that would be a welcome addition, I think